Dockerfile

```diff
@@ -5,7 +5,11 @@ WORKDIR /app
 
 COPY package.json /app/
 RUN npm install -g node-gyp && npm install --unsafe-perm
+RUN npm install
```

Comment:
why again after already running `npm install --unsafe-perm`?

Reply:
Missed that. Should be fixed now.